### PR TITLE
Add team/ring0/qemu-stable branch and fix the build

### PR DESCRIPTION
**Pull request by gaborigloi**
Signed-off-by: Gabor Igloi <gabor.igloi@citrix.com>

### CODE REVIEW

**Comment by gaborigloi** [comment]
I ran my version of `upload-scripts`, and I can confirm that something was indeed uploaded for the new branch.

File: Dockerfile [hunk 1]
```diff
@@ -1,14 +1,21 @@
...
+# update the opam-repository
+WORKDIR ./opam-repository
+RUN git pull
+RUN opam update
 
-WORKDIR tmp/
+
+# First copy the opam file, and install the build dependencies - this way we
+# will only install the build dependencies when the opam file changes.
+COPY opam /tmp/
```
Comment (Frezzle) [inline Dockerfile:16]: Nice! I like faster (re)builds :+1:

**Comment by Frezzle** [comment]
Everything looks good to me!

**Review by Frezzle — APPROVED** [review]

**Comment by Frezzle** [comment]
Corresponding xenserver-build-env PR: https://github.com/xenserver/xenserver-build-env/pull/85

**Pull request merged**